tdrs-backend/tdpservice/users/urls.py

```diff
@@ -0,0 +1,12 @@
+"""Routing for Users."""
```

Comment:
Moved user urls to the user app. This will be updated as more user functions are added later.